README.md

```diff
@@ -2,29 +2,6 @@
 A Microverse exercise to implement the hamberger of the mobile version
 <a name="readme-top"></a>
 
-<div align="center">
-  <h3><b>Microverse README Template</b></h3>
-</div>
-
-# 📗 Table of Contents
-
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [📝 License](#license)
-
```

Comment:
Great hob so far but I think you removed the table of content in your README but you shouldn't remove it,
I think you missunderstood the previous code reviewer so he was asking for changing the logo title and put you own logo or project logo! so kindly add it again! ✅💯🔥🙏🏻🙏🏻